lib/sendAudit.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+
+var dgram = require('dgram')
+var net = require('net')
+var tls = require('tls')
+
+var sendUDPAudit = function(msg, host, port, callback) {
+  var client = dgram.createSocket('udp4');
+
+  return client.send(msg, 0, msg.length, port, host, function(err) {
+    client.close();
+    return callback(err);
+  });
+};
+
+var sendTLSAudit = function(msg, host, port, options, callback) {
+  var client = tls.connect(port, host, options, function() {
+    if (!client.authorized) {
+      console.log(client.authorizationError)
+      return callback(client.authorizationError);
+    }
+    client.write(msg.length + " " + msg);
+    return client.end();
+  });
+  client.on('error', function(err) {
+    return callback(err)
+  });
+  client.on('close', function(had_error) {
+    if (!had_error) {
+      return callback();
+    }
+  });
+};
+
+var sendTCPAudit = function(msg, host, port, callback) {
+  var client = net.connect(port, host, function() {
+    client.write(msg.length + " " + msg);
+    return client.end();
+  });
+  client.on('error', function(err) {
+    return callback(err)
+  });
+  client.on('close', function(had_error) {
+    if (!had_error) {
+      return callback();
+    }
+  });
+};
+
+exports.sendAuditEvent = function(msg, connDetail, callback) {
```

Comment:
It would be nice to add some API docs for this on the README. So that people know how to use it, not essential if we run out of time though.